DataTables.Blazor/Options/DataTableOptions.cs

```diff
@@ -58,6 +58,9 @@ public class DataTableOptions
         [JsonPropertyName("dom")]
         public string DOM { get; set; }
 
+        [JsonPropertyName("buttons")]
```

Comment:
Do you need this attribute? The DOM has one as it's not a classic `camelCase` <-> `PascalCase` conversion

Reply:
Fixed; I prefer to be explicit with my attributes re: serialization, but not needed

Reply:
That's a fair stance in this case where datatables itself is really only expecting one type of casing. The only issue is if you serve up the options as JSON over an API in say PascalCase, and then deserialize them. Maybe I'll update all of the properties to add an attribute